run_blockchain_terraformer.py

```diff
@@ -694,12 +712,12 @@ def organize_pool_details_solidly_v2(
 
     is_stable = True if "stable" in stable_pool else False
 
-    if "velocimeter" in exchange:
+    if exchange in [VELOCIMETER_V2_NAME]:
         default_fee = factory_contract.caller.getFee(pool_address)
         default_fee = float(default_fee) / 10000
-    elif "scale" in exchange:
+    elif exchange in [SCALE_V2_NAME, EQUALIZER_V2_NAME]:
         default_fee = factory_contract.caller.getRealFee(pool_address)
-        default_fee = float(default_fee) / 10 ** 18
+        default_fee = float(default_fee) / 10 ** 18 if exchange == SCALE_V2_NAME else float(default_fee) / 10000
```

Comment:
Exact same code (used to appear) in solidly_v2.py; Why is it here too?

Reply:
@Lesigh-3100 ?

Reply:
@barakman I pushed changes to make the code in line with the solidly_v2.py exchange style.

Reply:
Why duplicate the code instead of reusing it?
Ideally, this module should just call a function (or functions) in the other module.

Reply:
The functions in terraformer are not async because the module does not use async to avoid rate limiting. We could consider restructuring it to use the exchange classes but it would be a major refactor.

Reply:
You don't need to call the function as is.
There is obviously a piece of code here duplicated in two places.
You can extract that piece of code into a separate function, and then use that function where needed.

Reply:
I refactored this to use the solidly_v2 exchange class's functions.